blog/releases/announcing-knative-v0-25-release.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "v0.25 release"
+linkTitle: "v0.25 release"
+Author: "[Carlos Santana](https://twitter.com/csantanapr)"
+Author handle: https://github.com/csantanapr
+date: 2021-08-22
+description: "Knative v0.25 release announcement"
+type: "blog"
+image: knative-eventing.png
+---
+
+
+### Announcing Knative v0.25 Release
+
+A new version of Knative is now available across multiple components.
+
+Follow the instructions in the documentation
+[Installing Knative](https://knative.dev/docs/admin/install/) for the respective component.
+
+#### Table of Contents
+- [Highlights](#highlights)
+- [Serving v0.25](#Serving-v025)
+- [Eventing v0.25](#Eventing-v025)
+- [Eventing Extensions](#Eventing-Extensions)
+    - [Apache Kafka Broker v0.25](#Apache-Kafka-Broker-v025)
+    - [Apache Kafka Source and Channels v0.25](#apache-kafka-source-and-channels-v025)
+- `kn` [CLI v0.25](#Client-v025)
+- [Thank you contributors](#Thank-you-contributors)
+
+
+### Highlights
+- In preparation for GA, the **net-kourier** components have been renamed. See [Breaking Changes](#-breaking-or-notable-changes) in the Serving section.
+- The depecreated namespace label `networking.internal.knative.dev/disableWildcardCert` is now removed. See [Breaking Changes](#-breaking-or-notable-changes) in the Serving section.
+- Trigger ready status now takes into account the `DeadLetterURI`
+- The `kn` CLI now looks for plugin in the `$PATH` allowing for easy installation of plugins like `kn-quickstart`
+
+
+
+
+### Serving v0.25
+
+<!-- Original notes are here: https://github.com/knative/serving/releases/tag/v0.25.0 -->
+
+#### 🚨 Breaking or Notable Changes
+
+- **Renaming of some `net-kourier` component**
+
+    Related issue: [knative/networking#448](https://github.com/knative/networking/issues/448)
+
+    As part of our efforts to GA/1.0 we've standardized on the naming of our networking plugins that
+    are installed alongside Serving. If you're managing your Knative deployment manually with
+    `kubectl` **this will require a two-phase upgrade process**. In order to upgrade [net-kourier to v0.25.0](https://github.com/knative-sandbox/net-kourier/releases/tag/v0.25.0) using `kubectl` please follow the steps:
+
+        ```bash
+        # Apply the new release
+        $ kubectl apply -f net-kourier.yaml
+
+        # Once the deployment is ready apply the same file but
+        # prune the old resources
+        $ kubectl apply -f net-kourier.yaml \
```

Comment:
where do I get or download this net-kourier.yaml? Perhaps it can be added to the paragraph before or use a URL

Reply:
Hi, I state above where to get the yaml https://github.com/knative-sandbox/net-kourier/releases/tag/v0.25.0 for `net-kourier.yaml` if you are upgrading then you are already familiar with the kourier installation since you did it once.